src/pathfinding_service/service.py

```diff
@@ -13,13 +13,17 @@
     create_channel_event_topics,
     create_registry_event_topics,
 )
+from raiden.constants import PATH_FINDING_BROADCASTING_ROOM
+# from matrix_client.errors import MatrixRequestError
```

Comment:
what about this

Reply:
this was meant for the next debugging release. deleted it for now